### DISCUSSION TITLE
局域网内其他设备访问

### DISCUSSION THREAD

**Discussion by 0Miku0831** [mikumifa/biliTickerBuy#91] [提问专区]
启动项目提示：“To create a public link, set `share=True` in `launch()`",。我有公网IP，ipv4,想通过端口转发的方式让朋友帮我点验证码，请问这个函数在哪里更改呢？

**Comment by mikumifa** [comment]
v2.7.2 解决， 使用gradio 自带的免费72小时的公网转发
